dist/debian/debian_files_gen.py

```diff
@@ -29,7 +29,7 @@ class DebianFilesTemplate(string.Template):
     product = f.read().strip()
 
 with open('build/SCYLLA-VERSION-FILE') as f:
-    version = f.read().strip().replace('.rc', '~rc').replace('_', '-')
+    version = f.read().strip()
```

Comment:
I suggest to drop this after we complete a cycle with the new names. It doesn't hurt if the names already have ~rc.

Also, why did you change the second .replace?

Reply:
Ok, we can actually wait. As for the second part, it's not being used since our version syntax is `5.1.dev` or `5.1.0~rc` , so there is no `_` in version string (this is the original commit https://github.com/scylladb/scylla/commit/9de65f26ded98fc2570951b680aea302b96d3eac)